rcl/test/rcl/test_node.cpp

```diff
@@ -751,6 +751,5 @@ TEST_F(CLASSNAME(TestNodeFixture, RMW_IMPLEMENTATION), test_rcl_node_names) {
 TEST_F(CLASSNAME(TestNodeFixture, RMW_IMPLEMENTATION), test_rcl_node_options_fail) {
   rcl_node_options_t not_ini_options;
```

Comment:
Since `not_ini_options` is uninitialized there is a possibility that the impl pointer could be 0 in some runs, so you might be better off explicitly setting it to a non-null value for this test.

Reply:
Precisely this @Blast545, not a good idea to use an uninitialized variable.

Reply:
I think this is why windows failed. It probably had not_ini_options.impl as NULL, either randomly or intentionally.

Reply:
@hidmic I agree that's a bad idea to leave `not_ini_options` uninitialized before calling the copy method. Just checked the docblock in the header and already says that the destination has to be initialized, so I can reformat the test.

@brawner @hidmic However, this PR is catching a failure for the "not init" scenario, which means that if the variable is init this check won't be catched for Line Coverage. Any Ideas how should I proceed?

Reply:
I think all you need to do is assign a pointer to not_ini_options.impl to catch the expected failure on copy.